tests/drivers/i2s/i2s_speed/src/test_i2s_speed.c

```diff
@@ -540,3 +497,44 @@ ZTEST(drivers_i2s_speed, test_i2s_dir_both_transfer_long)
 	}
 	zassert_equal(num_verified, NUM_BLOCKS, "Invalid RX blocks received");
 }
+
+void *test_i2s_speed_configure(void)
+{
+/** Configure I2S TX transfer. */
```

Comment:
Nit-please align these comments with the code, and change style from `/** */` to `/* */`